sky/execution.py

```diff
@@ -96,7 +100,7 @@ class Stage(enum.Enum):
 
 
 def _execute(
-    dag: sky.Dag,
+    dag_or_task: Union['sky.Dag', 'sky.Task'],
```

Comment:
On naming, some options I can see:

1. `dag_or_task: Union['sky.Dag', 'sky.Task']`
2. `task_or_dag: Union['sky.Task', 'sky.Dag']` (promotes the easier usage first)
3. `entrypoint: Union['sky.Task', 'sky.Dag']` 
4. `task: Union['sky.Task', 'sky.Dag']`

I'm leaning towards (3), because the argument name will read more intuitive and it no longer suggests the concept of a DAG. Wdyt?